apps/server/.env.example

```diff
@@ -15,6 +15,9 @@
 # AUTH_ENABLED=false
 # EGO_API=
 # SCOPE=
+# # CORS allowed origins can be a comma separated list of the allowed domains.
```

Comment:
does this parse spacing? e.g. `a,b,c` vs `a, b, c`

Reply:
no, this takes whitespace as part of the value. very simple implementation